### PR TITLE
Capstone Project Design

### PR DESCRIPTION
**Pull request by NickEmma**
### This is how i designed the webpage
- Added a github workflow
- Created a good file and folder structure
- Linked all files together 
- Displayed the link redirect between home page and about page
- added dynamic javascript to display the speakers
- Added smooth scroll behavior for website transition dropdown
- This the link to the loom video describing. https://www.loom.com/share/f9021a3d15af417491cdf8566ce5f08e

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,866 @@
...
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  padding: 2rem 9%;
+  background: rgb(212 208 208 / 9%);
+  backdrop-filter: blur(5px);
+}
+
+#menu {
```
Comment (arslanbisharat) [inline css/styles.css:41]: - [ ] According to Microverse guidelines, please avoid using id as a selector. ID selectors have a higher specificity than attribute selectors but using ids in css isn't recommended and can't be reused. CSS classes can do everything IDs can.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+          </li>
+        </ul>
+      </div>
+      <nav class="navbar">
+        <a href="index.html" class="logo">
+          <img src="./assets/images/logo.svg" alt="" />
+        </a>
+        <ul>
+          <li>
+            <a href="about.html">About</a>
```
Comment (arslanbisharat) [inline about.html:44]: - [ ] I can move from `Home `page to `About `page but if I have to move back to `Home `page from `About `page, I have to click on the logo. I would recommend you to please add a `Home `page button when someone is present on `About `page.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        <p class="footer-title">
+          2022 creative commons Portugal, all right reserved
+        </p>
+        <p class="copyright about-copyright">
+          CC Bootcamp Summer 2022 works under the Creative Commons license which
+          means that the content provided during the program will be under the
+          same license. Refer to the Creative Commons page to read more about
+          it.
+        </p>
+      </div>
```
Comment (arslanbisharat) [inline about.html:175]: - [ ] The footer at the Home page doesn't have dropdown icon as present in the design. 
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+        <h3>"Hello! future developer"</h3>
+        <h1 class="about-title about-header">
+          creative commons bootcamp summer 2022
+        </h1>
+        <hr class="line" />
+        <div class="about-text">
+          <p class="pfirst">
+            The Creative Commons Bootcamp Summer 2022 was born in April 2015,
+            not just as a programming foundation, a school, an Institute, or
+            anything else you can imagine. It was born as a proposition. As an
```
Comment (arslanbisharat) [inline about.html:73]: - [ ] The text layout doesn't meet the design provided by Microverse. 

Your Design: 
image

Original 
image

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @NickEmma, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 1 above)
Comment (NickEmma) [reply css/styles.css:41]: okay

**Comment by NickEmma** [comment]
I don't understand what you mean, like what you told me to do

**Comment by arslanbisharat** [comment]
> I don't understand what you mean, like what you told me to do
> […](#)

Thank you so much for reaching out. I believe my requested changes are very clear. I would be happy to assist you and explain it to you. Can you please address what you didn’t understand? 
looking forward to hearing from you

**Comment by NickEmma** [comment]
everything i don't understand everything do you've some time to spend like
10 - 15 minutes to explain on zoom if you don't mind

File: about.html (hunk 2 above)
Comment (NickEmma) [reply about.html:44]: i used the logo so that when the user clicks on the logo it should take them back to the home

File: about.html (hunk 3 above)
Comment (arslanbisharat) [reply about.html:175]: There should be a drop down icon present after More text as given in the design

File: about.html (hunk 2 above)
Comment (arslanbisharat) [reply about.html:44]: When we are on about page instead of About button it should have Home button

**Comment by NickEmma** [comment]
okay i have resolved the issue now i will put the icon on the more button

File: about.html (hunk 4 above)
Comment (NickEmma) [reply about.html:73]: i have changed it to the original color

File: css/styles.css (hunk 1 above)
Comment (zieeco) [inline css/styles.css:41]: **[REQUIRED CHANGES]**

- Hey, great job with your implementation, however, as required by the previous reviewer, kindly use a CSS class to style your `menu`, then use the id to target it using javascript. e.g ` <div id="menu" class="fas fa-bars menu-icon"></div>`

```
.menu-icon {/* put the style here*/ } or replace `#menu` with `.menu-icon`

File: about.html (hunk 2 above)
Comment (zieeco) [reply about.html:44]: @arslanbisharat, This is working correctly as when you click on the logo, it takes you back to the home page. 
**[OPTIONAL]**

- You could improve this ` <a href="index.html" class="logo">` to ` <a href="/" class="logo">` making it move to the home by default

File: about.html (hunk 3 above)
Comment (zieeco) [reply about.html:175]: - [x] This issue is well resolved :heavy_check_mark:

File: about.html (hunk 4 above)
Comment (zieeco) [reply about.html:73]: - [ ] This issue is well resolved :heavy_check_mark:

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi, @NickEmma ,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 1 above)
Comment (NickEmma) [reply css/styles.css:41]: Okay i have resolved this issue

**Comment by NickEmma** [comment]
@zieeco and @danmainah Thank you for reviewing and approving my code.

**Pull request merged**